src/date.rs

```diff
@@ -18,7 +18,8 @@ use crate::naive::{self, IsoWeek, NaiveDate, NaiveTime};
 use crate::offset::{TimeZone, Utc};
 use crate::DateTime;
 use crate::{Datelike, Weekday};
-
+#[cfg(feature = "rkyv")]
+use rkyv::{Archive, Deserialize, Serialize};
```

Comment:
Please put this in a separate block of imports, between the std/core block and the crate-internal imports. (Same for all the other imports.)

Reply:
It's not necessary for me to merge this PR, but if you don't mind doing the work that would be great! Please do it as a separate commit.

Reply:
done